src/containers/sb-file-uploader.jsx

```diff
@@ -42,6 +43,11 @@ const messages = defineMessages({
         id: 'gui.projectLoader.loadError',
         defaultMessage: 'The project file that was selected failed to load.',
         description: 'An error that displays when a local project file fails to load.'
+    },
+    uploadWarning: {
```

Comment:
`menu-bar.jsx` already has this message:
```
    confirmNav: {
        id: 'gui.menuBar.confirmNewWithoutSaving',
        defaultMessage: 'Replace contents of the current project?',
        description: 'message for prompting user to confirm that they want to create new project without saving'
    }
```
Could you pass that in instead?

Reply:
@benjiwheeler good idea, moved it to sharedMessages and used it in both places.